src/lib/mutect.ml

```diff
@@ -2,9 +2,27 @@ open Common
 open Run_environment
 open Workflow_utilities
 
+module Configuration = struct
+
+  type t = { name: string; parameters: (string * string) list; }
+
+  let to_json {name; parameters}: Yojson.Basic.json =
+    `Assoc [
+      "name", `String name;
+      "parameters",
+      `Assoc (List.map parameters ~f:(fun (a, b) -> a, `String b));
+    ]
+  let render {parameters; _} =
+    List.concat_map parameters ~f:(fun (a,b) -> [a; b])
+
+  let default =
+    {name = "default"; parameters = []}
+
+end
+
 let run ?(reference_build=`B37)
     ~(run_with:Machine.t) ~normal ~tumor ~result_prefix
-    ?(more_edges = []) how =
+    ?(more_edges = []) ~configuration how =
```

Comment:
I do have a default in the API exposed in pipeline.ml, so in practice you don't have to specify it, but left it required here as some of the other modules do (e.g. Strelka).